packages/react-composites/src/composites/MeetingComposite/styles/MeetingCompositeStyles.ts

```diff
@@ -6,4 +6,15 @@ import { IStackStyles } from '@fluentui/react';
 /**
  * @private
  */
-export const compositeOuterContainerStyles: IStackStyles = { root: { width: '100%' } };
+export const compositeOuterContainerStyles: IStackStyles = {
+  root: {
+    width: '100%',
+
+    // This allows the composite to correctly contain the call composite by calculating a height for the available space.
```

Comment:
<3 <3 <3 for this comment.

Can I frame this comment and put it up as required comments for all CSS that I ever approve for the rest of my life?